### ISSUE TITLE
Path mappings not working

### ISSUE THREAD

**Issue by jbrodriguez** [fathyb/parcel-plugin-typescript#3]
It still doesn't work for me. Any suggestions ?

package.json
```json
	"dependencies": {
		"flexboxgrid": "^6.3.1",
		"react": "^16.2.0",
		"react-dom": "^16.2.0",
		"react-redux": "^5.0.6",
		"react-router-dom": "^4.2.2",
		"redux-saga": "^0.16.0",
		"styled-components": "^2.2.4",
		"tachyons": "^4.9.0"
	},
	"devDependencies": {
		"@types/react": "^16.0.30",
		"@types/react-dom": "^16.0.3",
		"@types/react-redux": "^5.0.14",
		"@types/react-router-dom": "^4.2.3",
		"@types/redux-saga": "^0.10.5",
		"babel-preset-env": "^1.6.1",
		"babel-preset-react": "^6.24.1",
		"parcel-bundler": "^1.2.0",
		"parcel-plugin-typescript": "^0.2.0",
		"tslint": "^5.8.0",
		"typescript": "^2.6.2"
	}
```

tsconfig.json
```json
{
	"compilerOptions": {
		"target": "es2015",
		"module": "es2015",
		"jsx": "react",
		"allowSyntheticDefaultImports": true,
		"sourceMap": true,
		"noUnusedParameters": true,
		"noUnusedLocals": true,
		"noImplicitReturns": true,
		"noFallthroughCasesInSwitch": true,
		"baseUrl": "."
	},
	"include": [
		"src/**/*"
	],
	"exclude": [
		"node_modules"
	]
}
```
tree
```
- client
  - tsconfig.json
  - src
    - store.ts
    - app.tsx
```

app.tsx
```javascript
import React, { Component } from 'react'
import { render } from 'react-dom'

import { Store } from 'redux'
import { Provider } from 'react-redux'
import { BrowserRouter as Router, Route } from 'react-router-dom'

import getStore, { AppState } from 'src/store'
```

```bash
❯ npm start

> godupes@0.0.1 start /Volumes/Users/dev/code/src/testing/godupes/client
> parcel src/index.html

⏳  Building...
Server running at http://localhost:1234
🚨  /Volumes/Users/dev/code/src/testing/godupes/client/src/app.tsx: Cannot resolve dependency 'src/store'
    at /Volumes/Users/dev/code/src/testing/godupes/client/node_modules/browser-resolve/node_modules/resolve/lib/async.js:46:17
    at process (/Volumes/Users/dev/code/src/testing/godupes/client/node_modules/browser-resolve/node_modules/resolve/lib/async.js:173:43)
    at ondir (/Volumes/Users/dev/code/src/testing/godupes/client/node_modules/browser-resolve/node_modules/resolve/lib/async.js:188:17)
    at load (/Volumes/Users/dev/code/src/testing/godupes/client/node_modules/browser-resolve/node_modules/resolve/lib/async.js:69:43)
    at onex (/Volumes/Users/dev/code/src/testing/godupes/client/node_modules/browser-resolve/node_modules/resolve/lib/async.js:92:31)
    at /Volumes/Users/dev/code/src/testing/godupes/client/node_modules/browser-resolve/node_modules/resolve/lib/async.js:22:47
    at FSReqWrap.oncomplete (fs.js:152:21)
```

**Comment by fathyb** [comment]
I confirm this is a bug of the plugin, [I just released 0.2.2](https://github.com/fathyb/parcel-plugin-typescript/blob/develop/CHANGELOG.md) to fix this, can you try it?

**Comment by jbrodriguez** [comment]
Hi @fathyb, thanks for the quick reply !

It works if I set 

```json
  "target": "es2015",
  "module": "commonjs",
```

If i remove module or set it to something different, it fails, not sure why.

Any idea why ?

Bottom line, it's working now 👍

**Comment by fathyb** [comment]
Thanks for your feedback @jbrodriguez, I appreciate it. I found the cause of the first bug you described and fixed it in the `parcel-plugin-typescript@0.2.3-next.0` npm release. If you confirm it fixes the issue I will release it as `0.2.3`.

For the second bug [it's actually intended and it is a TypeScript behaviour](https://www.typescriptlang.org/docs/handbook/module-resolution.html#node). Just add `"moduleResolution": "node"` to your `compilerOptions` and it should instruct TypeScript to use Node.js module resolution (and so resolve the `index.ts`).
